pkg/cmd/experimental/syncgroups/rfc2307/ldapinterface.go

```diff
@@ -39,10 +40,11 @@ type LDAPInterface struct {
 	// groupQuery holds the information necessary to make an LDAP query for a specific
 	// first-class group entry on the LDAP server
 	groupQuery ldaputil.LDAPQueryOnAttribute
-	// GroupNameAttributes defines which attributes on an LDAP group entry will be interpreted as its' name
+	// GroupNameAttributes defines which attributes on an LDAP group entry will be interpreted as its unique identifier
```

Comment:
not sure how a chain of uid attributes would work in combination with `groupQuery.NewSearchRequest`, which expects the group uid to match a single attribute (or dn)